message_definitions/v1.0/messages.xml

```diff
@@ -2478,6 +2486,12 @@
       <field name="desired_sigma" type="int16">Desired angle to have w.r.t. the neighbor</field>
     </message>
 
+    <message name="KEY_EXCHANGE_GCS" id="159">
```

Comment:
what is the reason for not having the `link` attribute (probably set to `forwarded`) nor the `ac_id` field ?

Reply:
Good point. I must have missed that (I am using it with the Rust link so it worked as is). Will fix.

Reply:
Fixed